scalability_and_performance/ztp-zero-touch-provisioning.adoc

```diff
@@ -0,0 +1,28 @@
+// CNF-1500 assembly
+[id="ztp-zero-touch-provisioning"]
+= Zero touch provisioning
+include::modules/common-attributes.adoc[]
+:context: zero-touch-provisioning
+
+toc::[]
+
+Telco edge computing presents extraordinary challenges with managing hundreds to tens of thousands of clusters in hundreds of thousands locations. These challenges require fully-automated management solutions with, as closely as possible, zero human interaction.
```

Comment:
s/thousands locations/thousands of locations/...I think.

Reply:
Comment accepted.